docs/userGuide/plugins/texWithDollars.md

```diff
@@ -0,0 +1,40 @@
+### Plugin: TexWithDollars 
+
+This plugin allows you to use <tooltip content="$ ... $ or $$ ... $$">dollars</tooltip> as the delimiter for your math formulae.
+
+- Insert **inline** equations by enclosing them in single dollars `$...$`.
+
+- Insert **display** equations by enclosing them in double dollars `$$...$$`.
+
+- Insert numbered **display** equations by enclosing the equation in double dollars and the equation number in curly brackets `$$...$$ (1)`.
```

Comment:
```suggestion
- Insert numbered **display** equations by enclosing the equation in double dollars and the equation number in round brackets `$$...$$ (1)`.
```
I believe this should be "round" instead of "curly". Looks like a typo from earlier commits :)